wsmaster/che-core-api-workspace-activity/pom.xml

```diff
@@ -14,14 +14,13 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <artifactId>che-plugin-activity-parent</artifactId>
-        <groupId>org.eclipse.che.plugin</groupId>
+        <artifactId>che-master-parent</artifactId>
+        <groupId>org.eclipse.che.core</groupId>
         <version>6.3.0-SNAPSHOT</version>
-        <relativePath>../pom.xml</relativePath>
     </parent>
-    <artifactId>che-plugin-activity-wsmaster</artifactId>
+    <artifactId>che-core-api-workspace-activity</artifactId>
```

Comment:
It's a bit confusing to have `che-core-api-workspace-activity` and `che-core-workspace-activity-server` while they both have server-side code. Maybe there is a better naming approach to make clear that (as far as I understand) one should be packaged into wsmaster and another one into wsagent.